packages/table/src/regions.ts

```diff
@@ -691,3 +692,14 @@ export class Regions {
         return interval as ICellInterval;
     }
 }
+
+function intervalInRangeInclusive(interval: [number, number], minInclusive: number, maxInclusive: number) {
```

Comment:
The `this` binding on the `Regions` object is super brittle. I'd like to pull out `private` helper functions like these to remove the need for `this.***` altogether.

Reply:
unwinding `this` madness sounds great.

is there a type alias for `[number, number]`?

Reply:
YEP.

```tsx
export type ICellInterval = [number, number];
```